examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
```

Comment:
Missing "of":
"You may not use this file except in compliance with one OF these ..."